.github/workflows/kind.yml

```diff
@@ -27,69 +27,76 @@ jobs:
         # The "minimum" tag is set in the Makefile
         # KinD tags: https://hub.docker.com/r/kindest/node/tags
         kind:
-          - 'minimum'
-          - 'latest'
+          - "minimum"
```

Comment:
Could you please undo these style changes or at least put them in a separate commit?